content/ember-cli/v5/travis-ci-support.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Travis CI Support
+since: 5.4.0
+until: 6.0.0
+---
+
+Travis CI is no longer recommended.
```

Comment:
```suggestion
Travis CI is no longer recommended. Since Travis CI announced the end of its unlimited support for open-source projects, most of the Ember community has switched over to using GitHub Actions or other alternatives instead. 

The Ember CLI option for Travis CI when generating projects has been deprecated, including the `--ci-provider=travis ` command-line flag.

Travis CI may continue to be used, but the config will no longer be maintained or updated by the Ember CLI blueprint.
```